include/pstl/internal/parallel_backend_tbb.h

```diff
@@ -381,7 +381,7 @@ class merge_task: public tbb::task {
 const size_t __PSTL_MERGE_CUT_OFF = 2000;
 template<typename _RandomAccessIterator1, typename _RandomAccessIterator2, typename _RandomAccessIterator3, typename __M_Compare, typename _Cleanup, typename _LeafMerge>
 tbb::task* merge_task<_RandomAccessIterator1, _RandomAccessIterator2, _RandomAccessIterator3, __M_Compare, _Cleanup, _LeafMerge>::execute() {
-    const auto __n = (_M_xe-_M_xs) + (_M_ye-_M_ys);
+    const size_t __n = (_M_xe-_M_xs) + (_M_ye-_M_ys);
```

Comment:
What type of warning was in this line?
As I can see in common case the type of `(_M_xe-_M_xs)` should be `std::iterator_traits<_RandomAccessIterator1>::difference_type`
and type of `(_M_ye-_M_ys)` should be `std::iterator_traits<_RandomAccessIterator2>::difference_type`
What type of result should be?

Reply:
Comparison of `long` and `size_t`, due to the comparison done in the following line